src/aprl/configs/multi/score.py

```diff
@@ -272,85 +324,39 @@ def debug_two_agents(score):
-    def zoo_baseline():
-        """Try all pre-trained policies from Bansal et al's gym_compete zoo against each other."""
-        spec = {
-            'config': {
-                PATHS_AND_TYPES: tune.grid_search(_env_agents()),
-            },
-        }
-        exp_name = 'zoo_baseline'
-
-        _ = locals()  # quieten flake8 unused variable warning
-        del _
-
-    @multi_score_ex.named_config
-    def fixed_baseline():
-        """Try zero-agent and random-agent against pre-trained zoo policies."""
-        spec = {
-            'config': {
-                PATHS_AND_TYPES: tune.grid_search(_fixed_vs_victim('random') +
-                                                  _fixed_vs_victim('zero')),
-            }
-        }
-        exp_name = 'fixed_baseline'
-
-        _ = locals()  # quieten flake8 unused variable warning
-        del _
-
-    @multi_score_ex.named_config
-    def random_baseline():
-        """Try random-agent against pre-trained zoo policies."""
-        spec = {
-            'config': {
-                PATHS_AND_TYPES: tune.grid_search(_fixed_vs_victim('random')),
-            }
-        }
-        exp_name = 'random_baseline'
-
-        _ = locals()  # quieten flake8 unused variable warning
-        del _
-
-    @multi_score_ex.named_config
-    def adversary_transfer():
-        """Do adversarial policies trained on victim X transfer to victim Y?"""
-        spec = {
-            'config': {
-                PATHS_AND_TYPES: tune.grid_search(
-                    _adversary_vs_victims('ppo2', get_adversary_paths())
-                ),
-            }
-        }
-        exp_name = 'adversary_transfer'
+    # Standard experiments
 
-        _ = locals()  # quieten flake8 unused variable warning
-        del _
-
-    @multi_score_ex.named_config
-    def adversary_trained():
-        """Try adversaries against the victim they were trained against."""
-        spec = {
-            'config': {
-                PATHS_AND_TYPES: tune.grid_search(
-                    _adversary_vs_victims('ppo2', get_adversary_paths(), no_transfer=True)
-                ),
-            }
-        }
-        exp_name = 'adversary_trained'
+    @multi_score_ex.config
+    def default_placeholders():
+        spec = None
+        envs = None
+        victims = {}
+        opponents = {}
+        exp_prefix = {}
 
         _ = locals()  # quieten flake8 unused variable warning
         del _
 
-    @multi_score_ex.named_config
-    def summary():
-        """Try adversaries against the victim they were trained against."""
-        spec = {
-            'config': {
-                PATHS_AND_TYPES: tune.grid_search(_summary_paths()),
+    @multi_score_ex.config
+    def default_spec(spec, envs, victims, opponents, exp_prefix):
+        """Compare victims to opponents."""
+        if spec is None:
+            if not victims:
+                raise ValueError("You must specify the victims to compare with a modifier.")
```

Comment:
I found this a bit difficult to parse; suggested rephrase: "You must use a modifier config to specify victim policies you want to compare"  (also applies to opponent error below)